### PR TITLE
[FIX] portal: display avatar in row

### PR DESCRIPTION
**Pull request by papyDoctor**
Before: Avatar icon doesn't show up in task's portal list view
    
    Step to reproduce (runbot 16):
    - Install Project and 'portal' module
    - Create a task, set Joel Willis as customer and assign either Mitchell Admin or Marc Demo
    - Log in as portal
        Click on "Tasks"
        You should see a list view, with a column for assignees,
        with the inspector you can see the avatar icon is not displayed
    
    Now: Avatar icon is correctly displayed
    
    opw-3592972

---
I confirm I have signed the CLA and read the PR guidelines at www.odoo.com/submit-pr

### CODE REVIEW

**Review by nle-odoo — APPROVED** [review]
looks good to me, the "`.row > * { padding-right/padding-left:  calc(var(--gutter-x) * .5); }`" seems to be some generic things over row, and here it should not be applied :+1:

**Review by nboulif — APPROVED** [review]
LGTM, Thanks for the fix!
Hello @odoo/rd-sm-phishing , may I ask for a final check please? Thanks in advance!

**Review by reth-odoo — COMMENTED** [review]
I would say this doesn't deserve its own CSS rule. It's a specific issue with using a row over two elements that are already in a table, and are supposed to be stuck together.

I'd say a good fix would be to either get rid of the row, or add a wrapper element so the style of the row is applied to the two elements as a set instead of independently. Both are bad in stable though, so I'd say we use `px-0` on the problematic element directly, and then update the XML on master.

Note that for some reason we remove this in 17.0 but add it again on master, I'd say we can keep it. It was probably removed because of this issue in the first place.

tl;dr would prefer a boostrap class in stable, and updating the xml on master

Thanks for the fix!

**Comment by nboulif** [comment]
Closing this PR, will be fixed it here https://github.com/odoo/odoo/pull/147220
Thank you all!
